_plugins/hooks/cloudsearch.rb

```diff
@@ -0,0 +1,44 @@
+require 'json'
+enabled = ARGV.include?('--update-search-index')
```

Comment:
not familiar with `ARGV` but read up on it [here](https://stackoverflow.com/questions/13329132/whats-the-point-of-argv-in-ruby). Cool stuff